javascript/src/test/pages/Messages/SearchResults.test.js

```diff
@@ -46,7 +46,7 @@ describe("ChannelPage tests", () => {
 //             'data': [exampleMessage]
 //         });
 
-//         const { getByText } = render(<ChannelPage />);
+//         const { getByText } = render(<ChannelPageList />);

```

Comment:
I wonder whether this commented out code should be removed.

Is there a good reason why it is still here?  If not, then please remove it.  If so, let me know why.

Reply:
I have replaced those comments with working test cases.